tex/tutorial.tex

```diff
@@ -231,43 +218,30 @@ \section{Standard Library Functions}
 		Extend allows the reader to easily print to stdout. It is called with the \texttt{print} signature, and takes two parameters - the location to print to, and the string to print.
 
 		\begin{lstlisting}
-			print(1, "Hello World\n") \\ Prints Hello World to stdout
+			print(STDOUT, "Hello World\n") \\ Prints Hello World to stdout
```

Comment:
write(STDOUT, "Hello World")

Also, you might want to note somewhere, that it will only be invoked if it's value is requested, due to our lazy evaluation.